content/python/concepts/files/terms/rmdir/rmdir.md

```diff
@@ -50,3 +50,27 @@ os.rmdir("myfolder")
 # Delete folder in another directory
 os.rmdir("/path/to/myfolder")
 ```
+
+## Codebyte Example
+
+In this example the `./project` directory is deleted using the `.rmdir()` method:
+
+```codebyte/python
+# Imports os module
```

Comment:
I would remove the import os comment because it's not really adding any information

Reply:
remove on both entries